lib/irb/ruby-lex.rb

```diff
@@ -47,14 +44,19 @@ def self.compile_with_errors_suppressed(code, line_no: 1)
     result
   end
 
+  def check_command(code, context:)
```

Comment:
It returns boolean so name ends with `?` makes sense. 
How about just `command?` because rubocop Naming/PredicateName warns method name that starts with `is_`.

I also want to change `RubyLex#check_termination` to `terminated?`

```ruby
# Line 69~74
elsif check_command(code, context: context) # → is_command?(code, context: context) or command?(code, context: context)
  true
else
  code.gsub!(/\s*\z/, '').concat("\n")
  check_termination(code, context: context) # → terminated?(code, context: context)
end
```

Reply:
Yeah that sounds good 👍